### PR TITLE
Added plugin dropdown buttons

### PR DESCRIPTION
**Pull request by wrmk**

### CODE REVIEW

File: src/plugins/dropdown_buttons/plugin.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Selectize.define('dropdown_buttons', function () {
```
Comment (greenfork) [inline src/plugins/dropdown_buttons/plugin.js:1]: Can we move this plugin source code to our codebase? We are unlikely to merge this upstream, I don't think we want this change to be in the fork

**Review by greenfork — COMMENTED** [review]
I don't really know how this works, hopefully we will have this in Lookbook so I can tinker and see

**Comment by wrmk** [comment]
plugin moved to https://github.com/toughbyte/hub/pull/13341

**Comment by wrmk** [comment]
> plugin moved to [toughbyte/hub#13341](https://github.com/toughbyte/hub/pull/13341)

for now it is impossible to store plugin separate from selectize

**Pull request merged**